cypress/e2e/shared/about.test.ts

```diff
@@ -1,16 +1,15 @@
 import {Organization} from '../../../src/types'
 
 describe('About Page', () => {
-  beforeEach(() =>
-    cy.flush().then(() =>
-      cy.signin().then(() => {
-        cy.get('@org').then(({id}: Organization) => {
-          cy.visit(`/orgs/${id}/about`)
-          cy.getByTestID('about-page--header').should('be.visible')
-        })
-      })
-    )
-  )
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
```

Comment:
I intentionally chained these requests in [this PR](https://github.com/influxdata/ui/pull/2533) because it was causing flaky behavior

Reply:
that's not how cypress works though, these are all noops. they're not js promises, just promise... like. every time you call `cy.` or interact with a child chainable instance, you're immediately invoking a call to cypress' internal scheduler. it's why i can say `cy.visit(url); cy.get('selector').click();` and it'll wait until the page has rendered and updated and found the matching element before clicking, and also why annoying things like this exist, to re-engage with the scheduler's scope: https://github.com/influxdata/ui/blob/master/cypress/e2e/shared/telegrafs.test.ts#L274

Reply:
they hint at it in posts like: https://docs.cypress.io/guides/guides/debugging#Debug-just-like-you-always-do
but it's def not something that is intuitive

Reply:
right but I think the difference is that with `cy.flush` and `cy.signin` they perform requests with `cy.request`. And I think there are two ways to handle waiting for the response: 1) chaining with `.then` or 2) setting an alias and waiting with `cy.wait('@alias')`. Otherwise cypress will continue on because the 'promise' is not being handled. See this: https://github.com/cypress-io/cypress/discussions/9377#discussioncomment-142617

Reply:
oh you know what. Now that I refactored the `flush` and `signin` commands to return the cypress promise, you might actually be right that we no longer need to chain off of them https://docs.cypress.io/guides/references/best-practices#Unnecessary-wait-for-cy-request

Reply:
could we test this theory by making the change to not chain flush and signin in one test file on a new branch? That would trigger the CI to run that test 20x and could tell us if it will flake or not

Reply:
i dont understand what you're trying to say with #9377, that seems like it has more to do with the example i posted from `telegrafs.test.ts` than something specific about `cy.request`. I've run the flush / signin cycle a few hundred times while untangling some of these implicit timing couplings both locally and in CI (there's some legit things popping up while i'm refactoring), idk what another 20 would do?.

Reply:
cool if you've already run it a bunch of times and haven't seen flakes, then I'm fine with giving this a shot. I think you are correct that they don't need to be chained because the function now returns the cypress chainable promise. I just know for a fact that it did cause flaky behavior before and that's why I chained them all to begin with. But, in that same PR I refactored those functions which might have been all that was needed to fix the flakiness